packages/rocketchat-lib/server/startup/oAuthServicesUpdate.js

```diff
@@ -0,0 +1,103 @@
+/* globals CustomOAuth */
+const logger = new Logger('rocketchat:lib', {
+	methods: {
+		oauth_updated: {
+			type: 'info'
+		}
+	}
+});
+
+function debounce(fn, delay) {
```

Comment:
Because I think `_.debounce` doesn't use Meteor.setTimeout, if is not a problem...I can change